### PR TITLE
Turn on Issue filing feature flag

### PR DESCRIPTION
**Pull request by haonliu**
#### Description of changes

Turn on Issue filing feature flag. Do not merge until the other two PR(Azure Boards url & cancel button IX change) get in.

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Added relevant unit test for your changes. (`npm run test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`npm run precheckin`)
- [ ] Added screenshots/GIFs for UI changes.

### CODE REVIEW

File: src/common/feature-flags.ts [hunk 1]
```diff
@@ -69,10 +69,9 @@ export function getAllFeatureFlagDetails(): FeatureFlagDetail[] {
...
         {
             id: FeatureFlags.newIssueFilingExperience,
-            defaultValue: false,
+            defaultValue: true,
             displayableName: 'New issue filing experience',
-            displayableDescription:
-                'Enable the new issue filing experience. Needs "Issue filing" preview feature to be enable in order to work',
+            displayableDescription: 'Enable the new issue filing experience.',
             isPreviewFeature: false,
             forceDefault: true,
```
Comment (smoralesd) [inline src/common/feature-flags.ts]: I think you should change `forceDefault` to `false`, right?

Comment (haonliu) [reply src/common/feature-flags.ts]: Right, since we do not want the user to see the old (and broken) UX.

Comment (haonliu) [reply src/common/feature-flags.ts]: FYI, filing a bug from Target page is broken in the old UX.

Comment (waabid) [reply src/common/feature-flags.ts]: Force default should have been true before in all honesty; could've made it a bit easier to work on 😄

Comment (haonliu) [reply src/common/feature-flags.ts]: I'll get this in after Fer/Mark/Chelsea reviews the UX.

**Pull request merged**